databases/jdp/database.go

```diff
@@ -301,7 +270,62 @@ func dataResourceFromFile(file File) frictionless.DataResource {
 		Bytes:     file.Size,
 		Hash:      file.MD5Sum,
 		Sources:   sources,
-		Credit:    creditFromIdAndMetadata(id, file.Metadata),
+		Credit: credit.CreditMetadata{
+			Identifier:   id,
+			ResourceType: "dataset",
+			Titles: []credit.Title{
+				{
+					Language: "English",
+					Title:    filePath,
+				},
+			},
+			Dates: []credit.EventDate{
+				{
+					Date:  file.Date,
+					Event: "Created",
+				},
+				{
+					Date:  file.AddedDate,
+					Event: "Accepted",
+				},
+				{
+					Date:  file.ModifiedDate,
+					Event: "Updated",
+				},
+			},
+			Publisher: credit.Organization{
+				OrganizationId:   "JGI:JDP",
+				OrganizationName: "Joint Genome Institute",
+			},
+			RelatedIdentifiers: []credit.PermanentID{
+				{
+					Id:               file.Metadata.Proposal.DOI,
+					Description:      "Proposal DOI",
+					RelationshipType: "IsCitedBy",
```

Comment:
I will see if Sophie / Kayla have a relationship that they used for this and the award DOI below

Reply:
Thanks!